bazel/grpc_build_system.bzl

```diff
@@ -212,7 +212,7 @@ def grpc_cc_test(name, srcs = [], deps = [], external_deps = [], args = [], data
                 exec_compatible_with = exec_compatible_with,
             )
     else:
-        native.cc_test(tags = tags, **args)
+        native.cc_test(name = name, tags = tags, **args)
```

Comment:
it looks like this was wrong, because the corresponding test weren't being generated with the right name (trying to run the test resulted in "no such test" error). TBH I'm not 100% sure why this change helps, but it seems to be helping.